Module.yaml

```diff
@@ -0,0 +1,125 @@
+version: 1.0.0
+name: OPCUASubscribe
+description: OPC UA Subscribe connects to OPC UA Server and listens for incoming data messages and forwards the payload to next module.
+icon: https://icons-020-demo.s3.eu-central-1.amazonaws.com/mi_OPCUA_input.png
+type: process
+tags:
+  - OPC UA Subscribe
+  - process
+categories:
+  - id: category
+    name: Process
```

Comment:
We don't have a category `Process`. I would suggest `Policy` category.

Reply:
ok I can update it, but I believe we used process in the past with other modules.